setup.py

```diff
@@ -21,6 +28,11 @@ def c_files_in(directory):
             version = m.group(1)
 assert version
 
+class GPerfingBuildExt(build_ext):
+    def run(self):
+        os.system("gperf src/entities.gperf > src/entities.c")
```

Comment:
it's a bit odd that this part is hard-coded to `entities.gperf` but the other part finds any files with `.gperf`.